content/20.treatments.md

```diff
@@ -43,11 +43,9 @@ Perhaps, the TCZ treatment would best suit patients with severely compromised lu
 
 Antivirals
 
-Antivirals are an emerging category of drugs.
-Unlike antibiotics, they do not kill viruses.
-Rather, they inhibit the proliferation of a virus.
-Categories may include therapies that inhibit viral proteins, inhibit viral entry, and more
-Antivirals have been used to treat XYZ diseases through [what mechanisms or approaches are used?]
+Background: The replication cycle of a virus within an epithelial host cell includes 6 basic steps which can be shortly summarised as follow: i) attachment of the virus to the host cell; ii) penetration by endocytosis; iii) uncoating, classically defined as the release of viral contents into the host cell; iv) biosynthesis, during which the viral genetic material enters the nucleus where it gets replicated; v) assembly, where viral proteins are translated and new viral particles are assembled; vi) release, when the new viruses are released into the extracellular environment (ISBN-13: 978-1405136457). 
+
+Antiviral drugs do not kill the virus, rather they inhibit its amplification by impairing one of these steps. Nowadays, many of these drugs act during the biosynthesis step in order to inhibit the virus’ genetic material replication. Importantly, Corona viruses, including Covid-19, are RNA viruses. Differently from a DNA virus, which can use the host enzymes to propagate itself, RNA viruses depends on their own polymerase, the RNA-dependent RNA polymerase (RdRP), in order to be replicated (doi.org/10.1007/978-1-4939-2438-7; doi.org/10.1002/jmv.25761). 
```

Comment:
I would suggest deleting the "in order" in the last sentence so you end up with "the RNA-dependent RNA polymerase to be replicated"